infrastructure/ansible/roles/dataset_loader/tasks/dataset_loader.yml

```diff
@@ -1043,3 +1043,18 @@
   delay: 10
   vars:
     error_query: "alerts[?level=='error'].text[?!contains(@,'already exists')]"
+
+- name: Queue Caches in all CDNs
+  uri:
+    url: "{{ dl_to_url }}/api/{{ dl_to_api_version }}/cdns/{{ item.id }}/queue_update"
+    method: POST
+    body:
+      action: "queue"
+  with_items: "{{ get_all_cdns.json.response[1:] }}"
```

Comment:
This should use a different method like json_query to exclude the ALL CDN because the order of the json isn't guaranteed.

Reply:
The previous task, *Snapshot all CDNs*, uses this method to exclude the *ALL* CDN
https://github.com/apache/trafficcontrol/blob/a0fbd63083140e64d35d886e0cc87cc8b542aaa6/infrastructure/ansible/roles/dataset_loader/tasks/dataset_loader.yml#L1034-L1038

Maybe someone else can add `json_query` to both tasks in the future.

Reply:
I don't think that both tasks should use different methods for filtering out the *ALL* CDN, though, and changing the *Snapshot all CDNs* task is outside the scope of this PR.

Reply:
> because the order of the json isn't guaranteed.

Also, the sort order is always the same
https://github.com/apache/trafficcontrol/blob/19f3b2eafde573546221e767c77f283994fb2fa7/traffic_ops/traffic_ops_golang/cdn/cdns.go#L140

Reply:
Ok, so even if json itself isn't we're coercing it to be.  It's still a bug laying in wait should you name a CDN something alphabetically before ALL.  If we want to merge this and open more issues to fix that separately that's fine.